tools/install-powershell.ps1

```diff
@@ -382,7 +388,9 @@ try {
 
         $null = New-Item -ItemType Directory -Path $contentPath -ErrorAction SilentlyContinue
         if ($IsWinEnv) {
-            if ($UseMSI -and $Quiet) {
+            if ($UseMSI -and $architecture -eq "arm64") {
```

Comment:
Would you rather add support for all msix packages. x64, x86, arm32 and arm64?

Reply:
Since this file is rather large and complex I was trying to limit the changes to the bare minimum for ARM64 support. However, it does make sense to support all flavors on MSIX releases. I can take a look at adding that. Are you going to block this pull request without that change?

Reply:
You can add the support for other platforms in later PR.